Framework/PythonInterface/mantid/plots/datafunctions.py

```diff
@@ -382,7 +382,7 @@ def get_bin_indices(workspace):
     if is_range:
         return range(range_start + 1, range_end)
     else:
-        indices = np.where(np.isin(range(total_range), monitors_indices, invert=True))
+        indices = np.where(np.isin(range(total_range), monitors_indices, invert=True))[0]
```

Comment:
Do we need to be careful here? The api seems to have changed as [1.13](https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.where.html), we have this on Ubuntu/CentOS, said it *could* return a tuple but [1.18](https://numpy.org/doc/1.18/reference/generated/numpy.where.html?highlight=where#numpy.where) Windows/Mac says it just returns an array always.
I think we might be missing a unit test in the least as no tests were failing before?

Reply:
@martyngigg just tried a repl on this and 1.19 still returns a `tuple` despite the documentation stating it always returns an array. Maybe we should do a type check? Agree we need to add a test.

Reply:
Ok. If 1.19 still returns a tuple then I think we are safe to assume that. Can we just add a test?